### DISCUSSION TITLE
Autocomplete field does not work  [TWE 1.1.0]

### DISCUSSION THREAD

**Discussion by AlJoff** [mdbootstrap/TW-Elements#2342] [Support from community] [answered]
The first time I enter text into a field with autocomplete and select a value from the dropdown list, autocomplete stops working properly. 
After the first value selection, the dropdown list becomes invisible (you can see it in the video) and overlaps other interface elements. Also, when clicking on the field with autocomplete again, the dropdown list does not open.

I don't understand why this is broken. This is a project from the Tailwind Elements v 1.1.0 documentation, meaning I have not changed the source code.

https://github.com/mdbootstrap/TW-Elements/assets/146883182/51fd9e82-a8e1-45a2-aa12-933b2befaacf

Help please as I want to use a field with autocomplete in my student term paper

**Comment by juujisai** [comment]
I do not see the issue for the 1.1.0 version - checkout this snippet: https://tw-elements.com/snippets/tailwind/b-cylwik/6092439#html-tab-view.

Try creating a new app and see if the issue is still there.

**Reply by AlJoff** [reply]
This is the new project (app) from the documentation. I didn't change anything and ran it as is.
I followed all the steps from: https://tw-elements.com/docs/standard/getting-started/local-installation/

**Reply by AlJoff** [reply]
What am I doing wrong? Here's the full video from Local installation TW 1.1.0. 
https://www.youtube.com/watch?v=36bhRALOHfE

**Comment by Pahomk** [comment, marked as answer]
This is a **bug** in the **Tailwind Elements** framework. 
Tailwind Elements such as Autocomplete will NOT work when Windows animations are disabled.

You must have Windows animations enabled. Otherwise, Tailwind Elements will not work properly. This applies to any web site whose user interface is built on Tailwind Elements, not just your local projects. 

Enable Windows animations and see if the problem has been resolved.

To enable Windows animations follow these steps:
1) Enable all visual effects:
Go to: Settings -> System -> Advanced system settings -> Perfomance - Settings... -> Visual Effects. Set "Let Windows choose what's best for my computer". 
= OR =
Press Win+R and paste "SystemPropertiesAdvanced" and Perfomance - Settings... -> Visual Effects. Set "Let Windows choose what's best for my computer". 
![image](https://github.com/mdbootstrap/TW-Elements/assets/115905749/b839a1a3-1b92-4be3-96c5-5773fb1f592c)
![image](https://github.com/mdbootstrap/TW-Elements/assets/115905749/afa2e8ee-c91e-4758-8853-2a148ab954f7)

2) Enable animations
Go to: Settings -> Ease of Access -> Display -> Show animations in Windows - Turn ON

![image](https://github.com/mdbootstrap/TW-Elements/assets/115905749/06974c7d-516f-4abd-92f9-007bf25f8715)

After that the user interface built on Tailwind Elements will work without errors.

**Reply by AlJoff** [reply]
No way... I've been puzzling over it for 3 days.... 
ITS WORKS !!!

What about if, suddenly, someone uses TWE in real projects (not for studying)?  So if a real site uses the TWE framework and the user has animations disabled, the UI will be broken?

**Reply by juujisai** [reply]
Thanks, we'll check our components for the `motion-reduce` cases
